gui/wxpython/startup/utils.py

```diff
@@ -55,6 +58,58 @@ def get_possible_database_path():
     return path
 
 
+def create_possible_database_path():
+    """Create the standard GRASS GIS directory.
+
+    Create directory named grassdata in the standard location according to the platform.  
+    
+    Returns the new path as a string or None if nothing was found or created.
+    """
+    home = os.path.expanduser('~')   
+    candidates = []       
+    
+    # Candidate for independent "grassdata" in for Linux and macOS
+    if sys.platform.startswith('linux') or sys.platform.startswith('darwin'):
+        candidates.append(os.path.join(home, "grassdata"))
+    
+    # Candidates for independent "grassdata" in other dirs (Windows)
+    else:  
+        candidates.append(os.path.join(home, "Documents", "grassdata"))
+        
+        try:
+            # here goes everything which has potential unicode issues
+            candidates.append(os.path.join(home, _("Documents"), "grassdata"))
+        except UnicodeDecodeError:
+            # just ignore the errors if it doesn't work
+            pass
+    
+    # Create "grassdata" directory           
+    for candidate in candidates:                
+        try:
+            os.mkdir(candidate)
+            path = candidate
+            return path
```

Comment:
Here, you don't have to assign to a variable here, just return candidate right away.